### PR TITLE
Embedded YouTube videos in MooseDocs and stochastic_tools doco typos

### LINKED ISSUE #15637: Make training video more prominent
## Reason
Currently the training video is buried in text when we could embed it into the page. 

## Design
use `MooseDocs` media extension to add the video to the front page. 

## Impact
Improve user training experience on website

### PR DESCRIPTION
**Pull request by cticenhour**
- Adds capability to embed videos from YouTube using `!media youtube` in MooseDocs
- Fixes a few typos and file path errors in `stochastic_tools` documentation that were discovered in the implementation of the new MooseDocs feature. @aeslaughter and @zachmprince - is there a ticket number you'd like me to reference for this fix? 
- Adds the workshop youtube video to the front page of the MOOSE website. @loganharbour - can you allow the webinar video to be embedded in other pages? As of right now it just refuses to play, citing lack of permission, and links you to YouTube.

Closes #15637

### CODE REVIEW

**Comment by loganharbour** [comment]
Should be good to embed now.

**Comment by dschwen** [comment]
I don't see the video. Is embedding enabled on the video settings?

**Comment by loganharbour** [comment]
I don't either. And yes, it is - you can confirm by clicking share on the video itself. Embed is an option.

**Comment by cticenhour** [comment]
The new HTML added to the page looks like it's supposed to: 

```HTML
<div style="text-align:center;"><iframe width="560" height="315" src="http://www.youtube.com/embed/2tJwBsYaLaI" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="allowfullscreen"></iframe>
```
and the embed link is correct.....weird.

**Comment by cticenhour** [comment]
So I've built the website on rod and viewed it on my local machine. I can still see the video there. Looking around online, I'm starting to wonder if it has something to do with the use of HTTP in the URL, causing "non-secure content" from being displayed. Do either of your browsers suggest this? The fix here would probably be just switching to HTTPS. 

I'm wondering if it could also be some sort of firewall or server side restriction? Other than that I'm stumped at the moment.

File: modules/doc/content/upcoming_training.md [hunk 1]
```diff
@@ -2,4 +2,9 @@
...
+- The recording for the recent MOOSE Training Webinar (June 9 - 10, 2020) can be found below and on YouTube [here](https://youtube.com/watch?v=2tJwBsYaLaI). You can find in the description of the video a summary that includes the covered sections, the time in the recording associated with each section, and the archived slide content associated with each section. We look forward to hosting more virtual workshops in the future and would like to thank the participants for their time and for their questions!
+
+!media youtube youtube_embed_src=https://www.youtube.com/embed/2tJwBsYaLaI
+               id=training-webinar
+               prefix=Video
+               caption=MOOSE training webinar given on June 9 - 10, 2020.
```
Comment (aeslaughter) [inline modules/doc/content/upcoming_training.md:10]:
```suggestion
- The recording for the recent MOOSE Training Webinar (June 9--10, 2020) can be found below and on [YouTube](https://youtube.com/watch?v=2tJwBsYaLaI). You can find in the description of the video a summary that includes the covered sections, the time in the recording associated with each section, and the archived slide content associated with each section. We look forward to hosting more virtual workshops in the future and would like to thank the participants for their time and for their questions!

!media youtube youtube_embed_src=https://www.youtube.com/embed/2tJwBsYaLaI
               id=training-webinar
               prefix=Video
               caption=MOOSE training webinar given on June 9--10, 2020.
```

File: python/MooseDocs/test/content/extensions/media.md [hunk 2]
```diff
@@ -53,3 +53,8 @@ Idaho: [idaho-flag]
        latex_src=big_buck_bunny.jpg
        style=width:100%;
        caption=Big Bug Bunny is an open-source movie short.
+
+!media youtube youtube_embed_src=http://www.youtube.com/embed/2tJwBsYaLaI
+               id=training-webinar
+               latex_src=training-webinar.png
+               caption=MOOSE training webinar given on June 9 - 10, 2020.
```
Comment (aeslaughter) [inline python/MooseDocs/test/content/extensions/media.md:60]:
```suggestion
               caption=MOOSE training webinar given on June 9--10, 2020.
```

Comment (aeslaughter) [reply python/MooseDocs/test/content/extensions/media.md:60]: You will need to update the golds for these changes.

**Review by aeslaughter — DISMISSED** [review]
Thanks!

**Comment by loganharbour** [comment]
Sweet! Looks great!

YouTube channel logo might be slightly off center... I'll see if I can fix it.

**Comment by cticenhour** [comment]
It doesn't look off-center on my monitor, but since it's centered with respect to the frame that it's in, it could be off-center compared to something else...

**Review by aeslaughter — APPROVED** [review]

**Pull request merged**